app/services/finance/npq/participant_eligible_and_payable_aggregator.rb

```diff
@@ -15,18 +15,18 @@ def self.aggregation_types
 
     private
 
-      attr_accessor :cpd_lead_provider, :recorder, :course_identifier
+      attr_accessor :cpd_lead_provider, :recorder, :course_identifier, :statement
 
       def initialize(statement:, course_identifier:, recorder: ParticipantDeclaration::NPQ)
-        @statement = statement
-        @cpd_lead_provider = statement.cpd_lead_provider
-        @recorder          = recorder
-        @course_identifier = course_identifier
+        self.statement         = statement
+        self.cpd_lead_provider = statement.cpd_lead_provider
+        self.recorder          = recorder
+        self.course_identifier = course_identifier
       end
 
       def aggregate(aggregation_type:, event_type:)
         scope = recorder.public_send(self.class.aggregation_types[event_type][aggregation_type], cpd_lead_provider, course_identifier)
-        scope = scope.where(statement_id: statement.id)
+        scope = scope.where(statement_id: statement.id) unless statement.open?
```

Comment:
if you plan to not to use declarations against this statement till after the payment date. this coupled with fact with another change in this PR to use `ParticipantEligibleAndPayableAggregator`. will mean that the statement will change in value after the cutoff but before the payment data. as it will pick up on new eligible declarations that are not assigned to this statement.